.jenkins/pytorch/test.sh

```diff
@@ -139,7 +139,7 @@ if [ -n "$CIRCLE_PULL_REQUEST" ]; then
 fi
 
 test_python_nn() {
-  time python test/run_test.py --include test_nn --verbose --determine-from="$DETERMINE_FROM"
+  time python test/run_test.py --include test_nn --verbose -- --run-parallel=4
```

Comment:
Is this assuming we don't need the `--determine-from` flag anymore?

Reply:
Also would it be better to dynamically compute the value of `run-parallel` to a value that's based on the number of cores available?

Reply:
No, I just temporarily remove determine from to make sure test is run (because it will incorrectly skip the test right now for python only changes)
And completely agree with you, we should determine number of parallel runs based on the number of cores